xsd/siri_model/siri_facility-v2.0.xsd

```diff
@@ -721,4 +721,37 @@ Rail transport, Roads and road transport
    </xsd:sequence>
   </xsd:sequence>
  </xsd:group>
+<!-- ===Vehicle FACILITY====================================== -->
+<xsd:complexType name="TypeOfVehicleFuelStructure">
+	<xsd:annotation>
+		<xsd:documentation>Classification of Vehicle Energy type. See Transmodel TypeOfFuel</xsd:documentation>
+	</xsd:annotation>
+	<xsd:sequence>
+		<xsd:element name="FuelType" type="TypeOfVehicleFuelEnumeration">
+			<xsd:annotation>
+				<xsd:documentation>Values for Vehicle fuel type. +SIRI v2.1</xsd:documentation>
+			</xsd:annotation>
+		</xsd:element>
+		<xsd:element name="VehicleFuelTypeOther" type="TypeOfValueStructure" minOccurs="0">
+			<xsd:annotation>
+				<xsd:documentation>An arbitrary vehicle fuel classification. +SIRI v2.1</xsd:documentation>
+			</xsd:annotation>
+		</xsd:element>
+	</xsd:sequence>	
+</xsd:complexType>
+<xsd:simpleType name="TypeOfVehicleFuelEnumeration">
```

Comment:
Please add the new value "unknown" to the existing type "TypeOfFuelEnumeration" of the siri_journey_support schema and then remove this new (and now obsolete" simpleType "TypeOfVehicleFuelEnumeration".